docs/changelogs/CHANGELOG-2.25.md

```diff
@@ -0,0 +1,216 @@
+
+### Action Required
+
+- **ACTION REQUIRED:** VMware Cloud Director: Support for attaching multiple networks to a vApp
+    - The field `ovdcNetwork` in `cluster` and `preset` CRDs is considered deprecated for VMware Cloud Director and `ovdcNetworks` should be used instead ([#12996](https://github.com/kubermatic/kubermatic/pull/12996))
+- **ACTION REQUIRED:** KubeLB(EE): The prefix for the tenant namespaces created in the management cluster has been updated from `cluster-` to `tenant-`. The tenants will be migrated automatically to the new namespace, load balancers, and services. The load balancer IPs need to be rotated and previous namespace cleaned up ([#13093](https://github.com/kubermatic/kubermatic/pull/13093))
+- **ACTION REQUIRED:** For velero helm chart upgrade related change. If you use `velero.restic.deploy: true`, you will see new daemonset `node-agent` running in `velero` namespace. You might need to remove existing daemonset named `restic` manually ([#12998](https://github.com/kubermatic/kubermatic/pull/12998))
+- **ACTION REQUIRED:** For velero helm chart upgrade. If running node-agent daemonset along with velero, then following replacement should be made in the velero's values.yaml before proceeding with upgrade `velero.restic.deploy` with `velero.deployNodeAgent``velero.restic.resources` with `velero.nodeAgent.resources``velero.restic.nodeSelector` with `velero.nodeAgent.nodeSelector``velero.restic.affinity` with `velero.restic.affinity``velero.restic.tolerations` with `velero.nodeAgent.tolerations` ([#13118](https://github.com/kubermatic/kubermatic/pull/13118))
+- **ACTION REQUIRED:** [User MLA] If you had copied `values.yaml` of loki-distributed chart to further customize it, then please cleanup your copy of `values.yaml` for user-mla to retain your customization only ([#12967](https://github.com/kubermatic/kubermatic/pull/12967))
+- **ACTION REQUIRED:** [User MLA] Cortex chart upgraded to resolve issues for cortex-compactor and improve stability of user-cluster MLA feature. Few actions are required to be taken to use new upgraded charts ([#12935](https://github.com/kubermatic/kubermatic/pull/12935)):
+    - Refer to [Upstream helm chart values](https://github.com/cortexproject/cortex-helm-chart/blob/v2.1.0/values.yaml) to see the latest default values.
+    - Some of the values from earlier `values.yaml` are now incompatible with latest version. They are removed in the `values.yaml` in the current chart. But if you had copied the original values.yaml to customize it further, you may see that `kubermatic-installer` will detect such incompatible options and churn out errors and explain that action that needs to be taken.
+    - The memcached-* charts are now subcharts of cortex chart so if you provided configuration for `memcached-*` blocks in your `values.yaml` for user-mla, you must move them under `cortex:` block.
+
```

Comment:
Let's create a Highlights section here as well including:
- EE: Add KubeVirt ([#12851](https://github.com/kubermatic/kubermatic/pull/12851))
- Upstream Documentation and SourceURLs can be added to ApplicationDefinitions ([#13019](https://github.com/kubermatic/kubermatic/pull/13019))
- EE: Add k8sgpt operator to the Default Application Catalogue ([#13025](https://github.com/kubermatic/kubermatic/pull/13025))
- Add K8sGPT to the Webshell ([#6501](https://github.com/kubermatic/dashboard/pull/6501))
- Add new feature to create, restore and schedule backups for user cluster namespaces ([#6296](https://github.com/kubermatic/dashboard/pull/6296))
- Add new page to manage backup storage location for the cluster backup feature ([#6478](https://github.com/kubermatic/dashboard/pull/6478))
- Support for downloading backups from the UI ([#6521](https://github.com/kubermatic/dashboard/pull/6521))
- Add support for Edge provider ([#6502](https://github.com/kubermatic/dashboard/pull/6502))
- Display comments in application values ([#6510](https://github.com/kubermatic/dashboard/pull/6510))
- Add Support for Kubernetes 1.29 ([#12936](https://github.com/kubermatic/kubermatic/pull/12936))

Reply:
done